type *http.Client has no field or method CloseIdleConnections

```
$ go get github.com/gopasspw/gopass

# github.com/hashicorp/vault/vendor/github.com/hashicorp/go-retryablehttp
../../../go/src/github.com/hashicorp/vault/vendor/github.com/hashicorp/go-retryablehttp/client.go:447:17: c.HTTPClient.CloseIdleConnections undefined (type *http.Client has no field or method CloseIdleConnections)
../../../go/src/github.com/hashicorp/vault/vendor/github.com/hashicorp/go-retryablehttp/client.go:507:16: c.HTTPClient.CloseIdleConnections undefined (type *http.Client has no field or method CloseIdleConnections)
../../../go/src/github.com/hashicorp/vault/vendor/github.com/hashicorp/go-retryablehttp/client.go:538:16: c.HTTPClient.CloseIdleConnections undefined (type *http.Client has no field or method CloseIdleConnections)
../../../go/src/github.com/hashicorp/vault/vendor/github.com/hashicorp/go-retryablehttp/client.go:545:15: c.HTTPClient.CloseIdleConnections undefined (type *http.Client has no field or method CloseIdleConnections)
../../../go/src/github.com/hashicorp/vault/vendor/github.com/hashicorp/go-retryablehttp/client.go:554:14: c.HTTPClient.CloseIdleConnections undefined (type *http.Client has no field or method CloseIdleConnections)
```

Maybe README needs to suggest a newer version of Go? I'm `go version go1.10.4 linux/amd64`

https://github.com/gopasspw/gopass/blob/8c7b4a250052d401722b6cbf9308b23a3d3d5661/README.md#L65

--- a/README.md
+++ b/README.md
@@ -62,7 +62,7 @@ Please see [docs/features.md](https://github.com/gopasspw/gopass/blob/master/doc
 
 ## Installation
 
-If you have [Go](https://golang.org/) 1.10 (or greater) installed:
+If you have [Go](https://golang.org/) 1.12 (or greater) installed:
 
 ```bash
 go get github.com/gopasspw/gopass

